common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -146,27 +149,44 @@
             },
 
             // @TODO move into utils.coffee
+            getNameWidth: function(name) {
+                var test = $('<div>'),
```

Comment:
FYI, @andy-armstrong - I didn't know issues like this would make the safe template linter fail. Good argument for adding that ESLint plugin you found to `eslint-config-edx` so we can catch these kinds of things in your editor before it hits CI.